src/test/regress/input/gp_check_files.source

```diff
@@ -48,15 +48,20 @@ $$ LANGUAGE plpgsql;
 CREATE TABLESPACE checkfile_ts LOCATION '@testtablespace@';
 set default_tablespace = checkfile_ts;
 
+--
+-- Tests for orphaned files
+--
```

Comment:
Why did you move this title here? Creating tablespace is also used for orphaned files.